Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM sitdocker/geocity-base:v1.3.2.2021
+#FROM sitdocker/geocity-base:v1.3.2.2021
+FROM geocity-base:latest
```

Comment:
Please match a release. This introduces the risk of pulling an image that is not aligned with the python libs and crash the whole application.

Reply:
Indeed. It has been fixed by commit: https://github.com/nicolas-heigvd/geocity/commit/30d985ac2e9f0cacf428c5ad833b9c1d5de9d374